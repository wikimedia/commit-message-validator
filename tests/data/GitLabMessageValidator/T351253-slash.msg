Update function-schemata sub-module to HEAD (17c4936)

New changes:
 - `c17cdad` definitions: Add Z560/Invalid evaluation result error type
 - `17c4936` definitions: Add recursion limit error types Z572 for
   orchestrator and Z573 for evaluator

Bug: T349784 \
Bug: T350608
Change-Id: I39cb9e012d4f9edbc862bb39fccb44133764fdbc
