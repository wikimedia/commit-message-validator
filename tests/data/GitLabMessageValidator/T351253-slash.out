The following errors were found:
- Line 8: Bug: value must be a single phabricator task ID
